lxd/networks.go

```diff
@@ -1488,11 +1488,26 @@ func networkStateGet(d *Daemon, r *http.Request) response.Response {
 		return resp
 	}
 
-	name := mux.Vars(r)["name"]
+	networkName := mux.Vars(r)["name"]
 
-	state, err := resources.GetNetworkState(name)
-	if err != nil {
-		return response.SmartError(err)
+	projectName := queryParam(r, "project")
+	if projectName == "" {
+		projectName = project.Default
+	}
+
+	var state *api.NetworkState
+	var err error
+	n, networkLoadError := network.LoadByName(d.State(), projectName, networkName)
```

Comment:
I would put this and the associated interface changes up as a separate PR to separate them from the ovn specific changes. Makes reviewing easier.

Reply:
Done (#9817) and updated the PR description to include it.